apps/web/src/components/Composer/Actions/ReferenceSettings/index.tsx

```diff
@@ -31,24 +49,135 @@ const ReferenceSettings: FC = () => {
   const isMyFollowers = isFollowerOnlyReferenceModule && onlyFollowers;
   const isMyFollows = isDegreesOfSeparationReferenceModule && degreesOfSeparation === 1;
   const isFriendsOfFriends = isDegreesOfSeparationReferenceModule && degreesOfSeparation === 2;
+  const [followValue, setfollowValue] = useState(
+    isMyFollows
+      ? MY_FOLLOWS.value
+      : isMyFollowers
+      ? MY_FOLLOWERS.value
+      : isFriendsOfFriends
+      ? FRIENDS_OF_FRIENDS.value
+      : isEveryone
+      ? EVERYONE.value
+      : EVERYONE.value
+  );
```

Comment:
Can we move this to a function with switch condition? the readability is very low